app/main/forms/suppliers.py

```diff
@@ -139,3 +139,40 @@ class CompanyOrganisationSizeForm(Form):
     organisation_size = RadioField('Organisation size',
                                    validators=[InputRequired(message="You must choose an organisation size.")],
                                    choices=[(o['value'], o['label']) for o in OPTIONS])
+
+
+class CompanyTradingStatusForm(Form):
+    OPTIONS = [
+        {
+            'value': 'limited company (LTC)',
```

Comment:
Do we want to store the whole thing with the acronym as a value?  Or we could stick with the existing radio button values, ie just "public limited company", "limited company", "limited liability company", "limited liability partnership"?

Also, we've never had "public bodies" as an option before, so if we make everyone else with existing data "other" then some of these might have been "public bodies".... I guess that's a question for the migration though.

Reply:
We display this back to the user on the summary page, and have nothing else that would translate it back ala what the content loader does, so I think we do need to store the entire value so that we can present back to the user the same thing they selected.

Reply:
As for the migration, that's exactly the point I've raised on the scripts PR.